firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -129,8 +129,9 @@ def delete_from_index(sender, instance, **kwargs):
         existing_entry = get_object_from_index(
             ELASTICSEARCH_DICTIONARY_ENTRY_INDEX, instance.id
         )
-        index_entry = DictionaryEntryDocument.get(id=existing_entry["_id"])
-        index_entry.delete()
+        if existing_entry:
```

Comment:
Small change as the deletion tests were failing if a local ES instance was running.